pages/common/ansible-doc.md

```diff
@@ -0,0 +1,29 @@
+# ansible-doc
+
+> Display information on modules installed in Ansible libraries.
+> It displays a terse listing of plugins and their short descriptions.
+> More information: <https://docs.ansible.com/ansible/latest/cli/ansible-doc.html>.
+
+- List available action plugins (modules):
+
+`ansible-doc -l`
+
+- List available plugins from specific type:
+
+`ansible-doc -t {{plugin_type}} -l`
+
+- Show information of action plugin (module):
+
+`ansible-pull {{plugin_name}}`
+
+- Show information of a plugin from a specific type:
+
+`ansible-doc -t {{plugin_type}} {{plugin_name}}`
+
+- Show playbook snippet for action plugin (modules):
```

Comment:
Perhaps this will read better?
```suggestion
- Show the playbook snippet for action plugin (modules):
```

Also, what is the technical term used by the official docs here? Is there a difference between a plugin and a module?

Reply:
Ansible plugins and module main different is plugins will execute on master doing some work to extend ansible capabilty while the module is executed on the target (remote) machine to do action on the machine.

But action plugins is the special case, some module action (action that target the remote one) need to be done in the master server first. From [this](https://groups.google.com/g/ansible-project/c/wNSJ4g-f4F8/m/xlpijLcRNSUJ) Brian who is Senior Software Engineer at ansible describe action plugins as special type of module.